### ISSUE TITLE
write pcap output file for each flow 

### ISSUE THREAD

**Issue by mikewalshchicago** [the-tcpdump-group/tcpdump#825]
Hello,

Is there any way to have a single tcpdump process write multiple output files?  Ideally, I would like an option flag that would identify each flow (src ip:port or dst ip:port), and write it to its own .pcap automaticaly.  I understand this may be unwieldy and i/o intensive, so perhaps an alternative would be to write a separate pcap for each item in a bpf filter.  For example, I was using a filter such as 'tcpdump -i eth0 (dst 192.168.1.2 and port 15310) or (dst 192.168.1.2 and port 15310) -w %' the files that would be written would be 192.168.1.1.15310.pcap and 192.168.1.2.15310.pcap.  

I'm open to alternatives or a discussion on how to best include such a feature.  Looking forward to your reply!

**Comment by gvanem** [comment]
I don't think so, but take a look at TCPFlow; https://github.com/simsong/tcpflow

**Comment by mcr** [comment]
No, there is not.  I am doing some related work with a new main() function [new program name is pktdump] that may be able to handle what you want, but it won't be ready for 2-3 months.

**Comment by mikewalshchicago** [comment]
Thanks, Guys.  I downloaded tcpflow and played around with it for a while, but I wasn't able to get it to ouput UDP to files in .pcap format (sorry, I should have made it clear that I am interested in both tcp and udp data).  I was able to get it to output the payloads of tcp packets in ascii, but I'd much prefer that raw binary to be retained.  I'll fiddle with it a bit more but I dont think tcpflow is going to be the solution for my use case.
